roles/hello-world-apb/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+app_name: "hello-world-{{ 99 | random(seed=apb_id) | to_uuid }}"
```

Comment:
+1 I like this approach a lot

Reply:
I like `_apb_service_instance_id` better :)

Reply:
We should just make a filter for this, so you could do something like `{{ 'hello-world' | apb_uuid }}` or something. Would let us control how the names are generated as well, if we could do it deterministically based on the service instance id then idempotence would be easy as well.

Reply:
How do we make a filter for this? I like this idea a lot...service instance id is a uuid, so it is unique and preserves idempotence. My concern is making it so that when you run the apb without a broker it still works (with sane defaults or you providing the value).

Reply:
I would also really like it if we were able to untie the APB variables from the broker. By this I mean that `_apb_service_instance_id` limits that variable to being used only when the APB is associated with a service instance. If we could go to more generics like `apb_id` or having a filter `apb_uuid`, I would really be on board.